drivers/network/dd/dc21x4/media.c

```diff
@@ -492,6 +494,7 @@ MediaInitDefaultMedia(
                     break;
                 case MEDIA_100T4:
                     MiiAdvertising = MII_ADV_100T4 | MII_CR_SPEED_SELECTION;
+                    MiiControl = 0;
```

Comment:
I assume the following was intended:
```suggestion
                    MiiAdvertising = MII_ADV_100T4;
                    MiiControl = MII_CR_SPEED_SELECTION;
```

Reply:
Yes, the surrounding code in the other switch-cases looks a bit like that. @disean should know for sure.